gcp-observability/build.gradle

```diff
@@ -36,11 +36,18 @@ dependencies {
             ('com.google.auth:google-auth-library-credentials:1.4.0'),
             ('org.checkerframework:checker-qual:3.20.0'),
             ('com.google.auto.value:auto-value-annotations:1.9'),
-            ('com.google.http-client:google-http-client:1.41.0'),
-            ('com.google.http-client:google-http-client-gson:1.41.0'),
+            ('com.google.http-client:google-http-client:1.41.7'),
+            ('com.google.http-client:google-http-client-gson:1.41.7'),
             ('com.google.api.grpc:proto-google-common-protos:2.7.1'),
             ("com.google.cloud:google-cloud-logging:${cloudLoggingVersion}")
 
+    implementation project(':grpc-census'),
+            libraries.opencensus_contrib_grpc_metrics,
+            ("io.opencensus:opencensus-exporter-stats-stackdriver:${opencensusVersion}"),
```

Comment:
Would be good to define these in `libraries` in the top level `build.gradle`. Also are you sure these will have the same version as `${opencensusVersion}` ? That is in future we might need to use a different version for `exporter-*-stackdriver` from `${opencensusVersion}` - or may be not...

Reply:
Moved `libraries` and `project` dependencies to top level in `build.gradle`. 

They are the same at the moment. To account for a different exporter version in future, renamed the variable to `opencensusExporterVersion`. Let me know if  I am misisng something.

And I did verify the combination of following versions using a manual test:

- `opencensusVersion` in `build.gradle`: `0.28.0` and `opencensusExporterVersion` in `gcp-observability/build.gradle`:`0.28.0`
- `opencensusVersion` in `build.gradle`: `0.28.0` and `opencensusExporterVersion` in `gcp-observability/build.gradle`:`0.31.0`
- `opencensusVersion` in `build.gradle`: `0.31.0` and `opencensusExporterVersion` in `gcp-observability/build.gradle`:`0.31.0`